diagnostic.md

```diff
@@ -41,7 +44,16 @@ format described in the instructions). The page should have a title in the head
 and an unordered list (with three items) inside the body.
 
 <!-- your answer starts here -->
-
+```
+<html>
+<head>
+<meta></meta>
+<Title></Title>
+</head>
+<body>
+</body>
+</html>
+```
```

Comment:
Close - forgot the unordered list with three items. Also keep all tags (aka <title> here) lowercase.

```html
<html>
  <head>
    <title> A Web Page </title>
  </head>
  <body>
    <ul>
      <li>A</li>
      <li>B</li>
      <li>C</li>
    </ul>
  </body>
</html>
```